apps/studio/components/layouts/AuthLayout/AuthLayout.utils.ts

```diff
@@ -33,6 +35,26 @@ export const generateAuthMenu = (
         { name: 'Users', key: 'users', url: `/project/${ref}/auth/users`, items: [] },
       ],
     },
+    ...(authenticationShowSecurityNotifications
+      ? [
+          {
+            title: 'Notifications',
+            items: [
+              ...(authenticationEmails
+                ? [
+                    {
+                      name: 'Email',
+                      key: 'email',
+                      pages: ['templates', 'smtp'],
+                      url: `/project/${ref}/auth/templates`,
```

Comment:
Singular name (_Email_ not _Emails_) because you’re configuring the notification type “email”. This will make more sense when other (singular) options are added: “Push”, ”SMS”.

Goes to existing `url` for now.